drivers/wifi/esp32/src/esp_wifi_drv.c

```diff
@@ -734,6 +734,7 @@ static int esp32_wifi_status(const struct device *dev, struct wifi_iface_status
 	}
 
 	strncpy(status->ssid, data->status.ssid, WIFI_SSID_MAX_LEN);
+	status->ssid[WIFI_SSID_MAX_LEN - 1] = 0;
```

Comment:
This does not look right. The `status->ssid` is 32 + 1 characters long, so in order to write to the last byte, then you want to do `status->ssid[WIFI_SSID_MAX_LEN] = '\0';`

Reply:
You're right, but when I saw that other codes were not written like this, I was a little suspicious.